.github/workflows/build-test-deploy.yaml

```diff
@@ -27,25 +27,6 @@ jobs:
       - name: Fails in order to indicate that pull request needs to be marked as ready to review and unit tests workflow needs to pass.
         run: exit 1
 
-  test:
```

Comment:
We ought to just remove `Run linters`. Removing this action completely means not unit tests are run since we added `--tags=integration` to the `test-integration` make target.